docs/get-started.mdx

```diff
@@ -0,0 +1,231 @@
+    svg="linux" />
+  <InstallBox
+    to="#run-netdata-with-docker"
+    os="Run with Docker" 
+    svg="docker" />
+  <InstallBox
+    to="/docs/agent/packaging/installer/methods/packages"
+    os="Linux with .deb/.rpm packages" 
+    svg="linux" />
+  <InstallBox
+    to="/docs/agent/packaging/installer/methods/macos"
+    os="Install on macOS" 
+    svg="macos" />
+  <InstallBox
+    to="/docs/agent/packaging/installer/methods/kubernetes"
+    os="Deploy on Kubernetes" 
+    svg="kubernetes" />
+  <InstallBox
+    to="/docs/agent/packaging/installer/methods/manual"
+    os="Linux from Git" 
+    svg="linux" />
+</Install>
+
+<details>
+  <summary><strong>Need a different installation method, or use a different operating system? See all our options.</strong></summary>
+  <Install>
+    <InstallBox
+      to="/docs/agent/packaging/installer/methods/kickstart"
+      os="Linux with one-line installer/kickstart script" />
+    <InstallBox
+      to="/docs/agent/packaging/installer/methods/packages"
+      os="Linux with .deb/.rpm packages" />
+    <InstallBox
+      to="/docs/agent/packaging/installer/methods/kickstart-64"
+      os="Linux with static 64-bit binary" />
+    <InstallBox
+      to="/docs/agent/packaging/installer/methods/manual" 
+      os="Linux from Git" />
+    <InstallBox
+      to="/docs/agent/packaging/installer/methods/source"
+      os="Linux from source" />
+    <InstallBox
+      to="/docs/agent/packaging/installer/methods/offline" 
+      os="Linux for offline nodes" />
+    <InstallBox
+      to="/docs/agent/packaging/installer/methods/docker"
+      os="Docker" />
+    <InstallBox
+      to="/docs/agent/packaging/installer/methods/kubernetes"
+      os="Kubernetes" />
+    <InstallBox
+      to="/docs/agent/packaging/installer/methods/macos"
+      os="macOS" />
+    <InstallBox
+      to="/docs/agent/packaging/installer/methods/cloud-providers"
+      os="Cloud providers (GCP, AWS, Azure)" />
+    <InstallBox
+      to="/docs/agent/packaging/installer/methods/freebsd"
+      os="FreeBSD"
+      community />
+    <InstallBox
+      to="/docs/agent/packaging/installer/methods/pfsense"
+      os="pfSense" 
+      community />
+    <InstallBox
+      to="/docs/agent/packaging/installer/methods/synology"
+      os="Synology" 
+      community />
+    <InstallBox
+      to="/docs/agent/packaging/installer/methods/alpine"
+      os="Alpine" 
+      community />
+    <InstallBox
+      to="/docs/agent/packaging/installer/methods/freenas"
+      os="FreeNAS" 
+      community />
+  </Install>
+</details>
+
+## Install Netdata on Linux
+
+The **recommended** way to install Netdata on a Linux node (physical, virtual, container, IoT) is our one-line
+[kickstart script](/packaging/installer/methods/kickstart.md). This script automatically installs dependencies and
+builds Netdata from its source code.
+
+Copy the script, paste it into your node's terminal, and hit `Enter` to begin the installation process.
+
+<OneLineInstall />
```

Comment:
Let's make the "anonymous statistics" a link that links to the page that defines it. ( I think it's security, under a header).

Reply:
Got this going on the preview: https://deploy-preview-534--netdata-docusaurus.netlify.app/docs/get-started/#install-on-linux-with-one-line-installer-recommended